examples/platforms/nxp/linux-imx/aarch64.cmake

```
@@ -0,0 +1,31 @@
+set( CMAKE_SYSTEM_NAME Linux )
```

1. missing license header.
1. could you remove the additional space inside of brackets?
```suggestion
set(CMAKE_SYSTEM_NAME Linux)
```

---

Thank you for the comments.
For comments 1&2, please help to review the updated one.